keras/src/models/model_test.py

```diff
@@ -171,6 +173,18 @@ def test_functional_single_output(self, model_fn, loss_type):
         )
         self.assertListEqual(hist_keys, ref_keys)
 
+        reloaded_pickle = pickle.loads(pickle.dumps(model))
+
+        pred_reloaded = reloaded_pickle.predict(x)
+        pred = model.predict(x)
+
+        # self.assertAllClose fails for some dtypes, so we use np
```

Comment:
Is it just a matter of calling `convert_to_numpy()` first? What's the problem?

Reply:
updated to call np.array({tensors}) and this works